pennylane/pauli/pauli_arithmetic.py

```diff
@@ -485,6 +485,14 @@ def __iadd__(self, other):
 
         raise TypeError(f"Cannot add {other} of type {type(other)} to PauliSentence")
 
+    def __sub__(self, other):
+        """Subtract other PauliSentence, PauliWord, or scalar"""
+        return self + -1 * other
+
+    def __rsub__(self, other):
+        """Subtract other PauliSentence, PauliWord, or scalar"""
+        return -1 * self + other
+
```

Comment:
Are we also adding these to `PauliWord`?

Reply:
Oops, must have forgotten to add those! thanks!

Reply:
updated ✅